### ISSUE TITLE
`AttributeError` when detecting multi table metadata from a CSV

### ISSUE THREAD

**Issue by Raghuramrwttw** [sdv-dev/SDV#1507]
im getting error like
AttributeError: 'SingleTableMetadata' object has no attribute '_load_data_from_csv'

this is my code:
metadata.detect_table_from_csv(
    table_name='hotels',
    filepath='C:/Users/Admin/Desktop/pythonsdv/hotels.csv'
)
and this is the error im getting
AttributeError                            Traceback (most recent call last)
Cell In[4], line 1
----> 1 metadata.detect_table_from_csv(
      2     table_name='hotels',
      3     filepath='[C:/Users/Admin/Desktop/pythonsdv/hotels.csv](file:///C:/Users/Admin/Desktop/pythonsdv/hotels.csv)'
      4 )

File [c:\Users\Admin\Desktop\pythonsdv\env\lib\site-packages\sdv\metadata\multi_table.py:357](file:///C:/Users/Admin/Desktop/pythonsdv/env/lib/site-packages/sdv/metadata/multi_table.py:357), in MultiTableMetadata.detect_table_from_csv(self, table_name, filepath)
    355 self._validate_table_not_detected(table_name)
    356 table = SingleTableMetadata()
--> 357 data = table._load_data_from_csv(filepath)
    358 print(data)
    359 table._detect_columns(data)

AttributeError: 'SingleTableMetadata' object has no attribute '_load_data_from_csv'

please help to get out from this error

**Comment by npatki** [comment]
Hi @Raghuramrwttw thanks for filing this issue. I'm going to classify this as a bug since I'm able to replicate it with the demo data.

### Workaround
A simple workaround is to load the data yourself as a pandas dataframe. Then you can use the other function [`detect_table_from_dataframe`](https://docs.sdv.dev/sdv/multi-table-data/data-preparation/multi-table-metadata-api#creation-api).

_Note that pandas is already pre-installed when you install SDV._

```python
import pandas as pd

hotels_dataframe = pd.read_csv('C:/Users/Admin/Desktop/pythonsdv/hotels.csv')
metadata.detect_table_from_dataframe(
    table_name='hotels',
    data=hotels_dataframe
)
```

You can do this independently for each table in your database.

### For SDV Developers
Seems like [`table._load_data_from_csv`](https://github.com/sdv-dev/SDV/blob/master/sdv/metadata/multi_table.py#L357) was left there accidentally. Shouldn't we be using the function from `utils` instead?

```
from sdv.utils import load_data_from_csv
```

It would also be good to add an integration test for this.

**Comment by amontanez24** [comment]
closed by #1533
